cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelperTest.java

```diff
@@ -61,6 +61,15 @@ private void createTopics() {
     }
   }
 
+  private void setWildcardThrottleReplicaForTopic(KafkaZkClient kafkaZkClient, String topicName) {
```

Comment:
Can we make this function `static`? This would make it clear that it does not modify the state of the object and just a utility function.

Reply:
OK. Btw, I used to add `static` to methods that are "stateless" in some other projects as well. But I was told by code reviewers not to do that and their argument is that `static` should, in general, be avoided as much as possible.

Reply:
I typically choose `static` to improve clarity if at all possible.
There are some cases, where mocking a method is needed and it is either not possible or tricky to do so with `static` methods (e.g. due to limitations of `EasyMock`). For those cases, making the method non-static would be understandable.